exercises/README.md

```diff
@@ -0,0 +1,98 @@
+# Programming exercises
+
+This directory contains the programming exercises
+and their autograder scripts together.
+
+## Installation of the client environment
+
+Install virtualenv. The command may differ depending on the system.
```

Comment:
To be clear, this is the requirement for students, right?

At this moment, it's okay.
But I think we need to check what are the typical environments and non-programming computer skills of our target students.

I feel `pip`, `pip3`, `virtualenv`, `./env/bin/activate` are overwhelming even for experienced non-Python programmer.

Reply:
At this moment, this instruction is for people who want to develop on this project,
but you are right that we should also mind the requirements and setup for the student-side installation.

I will look up the current setup used by the UoT course and will try to replicate that as close as possible,
and update the README later.